kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -360,3 +362,58 @@ def accounts(accts: list[KInner]) -> KInner:
             else:
                 wrapped_accounts.append(acct)
         return build_assoc(KApply('.AccountCellMap'), KLabel('_AccountCellMap_'), wrapped_accounts)
+
+    def pretty_print(self, kast: KAst, *, unalias: bool = True) -> str:
+        if unalias and isinstance(kast, KInner):
+            kast = undo_aliases(self.definition, kast)
+        if isinstance(kast, KInner):
+            kast = KEVM._int_token_to_hex(kast)
```

Comment:
My guess is this only does the rewriting if `kast` itself is `KInner`, but it won't recur into the `KInner` descendants of a `kast` that is not `KInner`. Is this intentional?

Also, are you sure you want to override the default `pretty_print`? This might result in a string that cannot be parsed back. (E.g. `"0x..."` is not a valid `Bytes` token.)

Reply:
Hmm, I thought `set_cell(kast, cell, bottom_up` would get to all descendants. Launching the debugger displays all the values as intended.